analysis/cox_model.do

```diff
@@ -193,13 +195,13 @@ format follow_up_end %td
 centile age, centile(10 50 90)
 mkspline age_spline = age, cubic knots(`r(c_1)' `r(c_2)' `r(c_3)')
 
-*save "output/`cpf'.dta", replace
-save "output/test.dta", replace
+save "output/`cpf'.dta", replace
+*save "output/test.dta", replace
 
 foreach var of varlist expo_hosp expo_non_hosp {
 	
-	use "output/`cpf'.dta", clear
-* 	use "output/test.dta", clear  // used on local system
+*	use "output/`cpf'.dta", clear
+ 	use "output/test.dta", clear  // used on local system
```

Comment:
Needs to be reverted for the real data run but we are not expecting the code to get there this time! 

In future, I would just set the local variable cpf to 'test' at the top of the script rather than changing multiple things later.